### PR TITLE
bpo-37004: Documented asymmetry of string arguments in difflib.SequenceMatcher for ratio method

### PR DESCRIPTION
**Pull request by sweeneyde**
https://bugs.python.org/issue37004

Automerge-Triggered-By: @tim-one

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

File: Doc/library/difflib.rst
Comment (tim-one) [inline Doc/library/difflib.rst]: No need to scream "not", and more accurate to say it may not be symmetric.  But rather than the high-falutin' "symmetric", I'd prefer plain language.  So, e.g.,

> Caution: The result of a :meth:`ratio` call may depend on the order of the arguments.  For instance::

File: Doc/library/difflib.rst
Comment (tim-one) [inline Doc/library/difflib.rst]: Long floating-point values distract from the point.  Suggest using the words "diet" and "tide" instead.  With "diet" first, the ratio is 0.5; with "tide" first, it's 0.25.

**Review by tim-one — CHANGES_REQUESTED** [review]
I like the idea!  But want some changes, as spelled out in comments.  Thanks!

**Comment by sweeneyde** [comment]
I have made the requested changes; please review again.

**Review by tim-one — APPROVED** [review]

File: Doc/library/difflib.rst
Comment (tirkarthi) [inline Doc/library/difflib.rst]:
```suggestion

```

Please remove the trailing space in this line causing CI error.

**Comment by sweeneyde** [comment]
What is the process for getting this PR merged?

**Comment by tim-one** [comment]
Thanks for speaking up!  While the change is fine by me, I'm not familiar enough with current workflow to finish it.  So I just posted a message to the python-dev mailing list, asking for someone else to proceed:

https://mail.python.org/archives/list/python-dev@python.org/thread/TFTQCJ7IZNCE3XUWEC4IVHPXBDZIKVEI/

I bet they'll ask that you add your name to the `Misc/ACKS` file.

**Comment by terryjreedy** [comment]
@tim-one The backports have to be 1. approved by a coredev, after briefly checking that done correctly, in which case the bot will merge if and when CI passes, or 2. directly merged by human, after CI passes.  I did the latter for 3.8 and 3.7.

**Comment by tim-one** [comment]
Thanks, Terry!  I was just going to kill them, to avoid screwing things up worse ☹️ 

While you're here, do you understand what the earlier

> Sorry, I can't merge this PR. Reason: Base branch was modified. Review and try the merge again..   

messages were trying to communicate?

And, when you did the merges, do you understand what the

 > Please replace # with GH- in the commit message next time. Thanks!

messages were trying to say?  I've seen the latter message every time I've "squashed & merged" anything, and still have no idea what it's talking about (I didn't use "#" in the commit messages).

**Comment by terryjreedy** [comment]
1. Good thing you left them alone.  They were fine.  If closed, new backport labels here would have been needed.  I open this and took a look at them because your fairly detailed explanation of what you did omitted mention of approving the backports.

2. Once CI for a patch to master passes, changes to master can create merge conflicts.  When so, Github adds a temporary message to the CI detail box.  (It is now shrunken, with a [View Details] button.)  One can resolved them by editing files with conflict markers either with Github's  online editor or with any editor on one's working machine, followed by a push of the edited branch.

When you added the Automerge label, our backport/merge bot, Miss Islington gets involved.  I resume its message paraphrases what I said above.  But I am puzzled because there is no listed merge-resolution commit.  Perhaps there was a spurious conflict that disappeared, perhaps github retried with a different and more successful merge strategy, perhaps there was just a timing issue.  Automerge is fairly recent, and I don't use it for changes to master because it hides the human who made the decision, and I often decide to merge after a final review.

3. The title at the top of this page has the PR number, here #13482, at the end, in gray.  It cannot be edited.  When one presses the green [Merge] button, github adds the number in ()s, such as (#13482) in black, and editable.  We are supposed to remember to change '#' to 'GH-', because #13482 could also refer to a tracker (bpo) number.  Automerge does that.  (What I don't like about automerge for original PRs is that it hides who triggered the merge.)

The same change is requested for backport numbers on backport merges (although there is discussion that they should maybe be deleted instead, or maybe the original PR #).  I forgot.

A coredev approving a backport is equivalent to an automerge label.  Then the change is automatic.  I could have used that instead.  I have not checked yet whether the label is available.

**Comment by tim-one** [comment]
Thanks again, Terry.  Ya, I assumed the "can't merge this PR" messages were complaining about a merge conflict, but didn't see any evidence of such.  Plus there were _three_ such messages, two of which _followed_ the bot saying it merged the commit into master.

So then I thought "OK, 3 complaints, but nothing fishy about master, but I added 3 backport labels so it must be complaining about them".  But no sign of merge conflicts in those either.

For the rest, sounds like I'll avoid automerge in the future.  It's hard enough trying to guess the consequences of what I explicitly do :wink:.

**Pull request merged**
